From 9a6f659803699dd20529c30a807e8f7853edb950 Mon Sep 17 00:00:00 2001
From: "Martin K. Petersen" <martin.petersen@oracle.com>
Date: Tue, 21 Apr 2015 20:06:10 -0400
Subject: [PATCH] oracleasm: 4.0 compat changes
Patch-mainline: Never, only needed by SLE12-SP2 oracleasm KMP
References: bsc#962720

mmarek: Cherry-picked from commit 882590a63c9a ("oracleasm: 4.0 compat
changes") in
git://git.kernel.org/pub/scm/linux/kernel/git/mkp/linux.git. The export
is needed by the oracleasm KMP. I changed it to _GPL, since mainline is
not exporting the symbol.

Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Michal Marek <mmarek@suse.com>
---
 block/bio.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/block/bio.c b/block/bio.c
index 13f51f62430c..d0f9d45f75f4 100644
--- a/block/bio.c
+++ b/block/bio.c
@@ -1364,6 +1364,7 @@ struct bio *bio_map_user_iov(struct request_queue *q,
 	bio_put(bio);
 	return ERR_PTR(ret);
 }
+EXPORT_SYMBOL_GPL(bio_map_user_iov);
 
 static void __bio_unmap_user(struct bio *bio)
 {
-- 
2.1.4

